cmd/driver/config/config.go

```diff
@@ -190,7 +190,7 @@ func checkFalcoRunsWithDrivers(engineKind string) error {
 }
 
 func (o *driverConfigOptions) replaceDriverTypeInFalcoConfig(hostRoot string, driverType drivertype.DriverType) error {
-	falcoCfgFile := filepath.Join(hostRoot, "etc", "falco", "falco.yaml")
+	falcoCfgFile := hostRoot + "/etc/falco/falco.yaml"
```

Comment:
```suggestion
	falcoCfgFile := filepath.Clean(filepath.Join(os.PathSeparator, hostRoot, "etc", "falco", "falco.yaml"))
```

Reply:
I am not sure what would then happen if you pass eg: `.` as hostRoot; we would end up with `/./etc/falco/falco.yaml` that is != than `./etc/falco/falco.yaml`.

Reply:
(this is what we do in tests btw)

Reply:
Also, we use string concatenation with hostRoot everywhere hostRoot is used in `pkg/driver`. I would go in with this PR and eventually change all of them to an unified method (perhaps a shared util function?) in a new one.

Reply:
ok got it.
I don't like too much the idea of using string concatenation in paths. But, I will be ok with this if there's no alternative. @alacuku, any idea? :thinking:

Reply:
> I am not sure what would then happen if you pass eg: `.` as hostRoot; we would end up with `/./etc/falco/falco.yaml` that is != than `./etc/falco/falco.yaml`.

Whether `hostRoot` is defined or not we should always get an absolute path. The problem here is in the tests where we are not using an absolute path. I would change the tests in this case, and make sure that `hostRoot` has a valid value before joining the path.

Reply:
> and make sure that hostRoot has a valid value before joining the path.

It is better to introduce an utility function and use it everywhere then; ok i'll do it!